src/main/java/com/github/onsdigital/babbage/content/client/ContentClient.java

```diff
@@ -115,7 +115,7 @@ public ContentResponse getContent(String uri) throws ContentReadException, IOExc
      *                              all other IO Exceptions are rethrown with HTTP status 500
      */
     public ContentResponse getContent(String uri, Map<String, String[]> queryParameters) throws ContentReadException {
-        return resolveMaxAge(uri, sendGet(getPath(DATA_ENDPOINT), addUri(uri, getParameters(queryParameters))));
+        return sendGet(getPath(DATA_ENDPOINT), addUri(uri, getParameters(queryParameters)));
```

Comment:
Do we know if this change has any impact elsewhere?

Reply:
Not for certain. but I have tested a number of pages that seem to still work that are served by Babbage and if the code is truthful this should only affect 'content' pages rather than things like downloads etc. - 

if cache is set to disable then the `resolveMaxAge` just returns instantly the same thing that sendGet returns. (but don't know last time chache enabled flag was set to false.

Also can't set the cache to false as search uses this too.